commons/src/main/java/org/dhis2/commons/orgunitselector/OUTreeFragment.kt

```diff
@@ -165,7 +169,9 @@ class OUTreeFragment private constructor() : DialogFragment() {
     private fun showAsDialog() = arguments?.getBoolean(ARG_SHOW_AS_DIALOG, false) ?: false
 
     private fun confirmOuSelection() {
-        selectionCallback(presenter.getOrgUnits())
+        singleEventEnforcer.processEvent {
+            selectionCallback(presenter.getOrgUnits())
```

Comment:
Same here, I think this singleevent enforcer should be handled from presentation logic

Reply:
Here I believe that this should not be migrated to the presenter, since all the orgtree presenter is doing is return the selected org units, and this way if we trigger the callback from here, it will apply for all places the orgUnitDialog is called, otherwise we will have to adapt it in all the implementations.  Let me know if this makes sense to you

Reply:
Ok, I have checked the code. I think this is the unique usage apart from the testing, right?.